src/main/kotlin/io/zerobase/smarttracing/resources/OrganizationsResource.kt

```diff
@@ -51,28 +52,31 @@ data class SiteResponse(val id: String, val name: String)
 @Path("/organizations")
 @Consumes(MediaType.APPLICATION_JSON)
 @Produces(MediaType.APPLICATION_JSON)
-class OrganizationsResource(val dao: GraphDao, private val siteTypes: MultiMap<String, String>, private val scanTypes: List<String>, private val amazon: Amazon) {
+class OrganizationsResource(val dao: GraphDao,
+                            private val siteTypes: MultiMap<String, String>,
+                            private val scanTypes: List<String>,
+                            private val email: Email) {
 
     @POST
     @Creator
     @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification = "false positive")
     fun createOrganization(request: CreateOrganizationRequest): IdWrapper? {
         val name = request.name
         val phone = request.contact.phone
-        val email = request.contact.email
+        val to_email = request.contact.email
```

Comment:
a) style: `camelCase` not `snake_case`
b) naming: `organizationEmail` not `toEmail`